scrapers.js

```diff
@@ -1692,14 +1692,13 @@ let scrapers = [
   {
     state: 'OH',
     country: 'USA',
-    url: 'https://odh.ohio.gov/wps/portal/gov/odh/know-our-programs/Novel-Coronavirus/welcome/',
+    url: 'https://coronavirus.ohio.gov/wps/portal/gov/covid-19/',
```

Comment:
```suggestion
```
Remove the URL from here

Reply:
Now that you're setting the URL below, you can accept this suggestion and remove it from above.